exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,8 @@
 const name = " Daniel  ";
 
-console.log(message);
+let greetingStart = "Hello, my name is ";
+
+let greeting = greetingStart + name.trim();
+let nameLength = name.length;
```

Comment:
What if `name` has whitespace characters round it, will this accurately report the length of the actual name itself?